src/dataflow/src/sink/kafka.rs

```diff
@@ -261,12 +296,13 @@ where
             encoder,
             producer,
             activator,
+            txn_timeout: Duration::from_secs(3),
```

Comment:
how did we land on this number? i know sometimes kafka clusters especially in CI can get overloaded so not sure if this is conservative enough / maybe this should be configurable?

Reply:
Completely made up. 🙂 
I was intending to make this configurable when I wire up a way to set the transactional.id, but it sounds like a good idea to at least bump up the default now